src/main/java/http/routes/EchoBody.java

```diff
@@ -0,0 +1,25 @@
+package http.routes;
+
+import http.Codes;
+
+import java.util.List;
+
+public class EchoBody {
+    final String CRLF;
+    final List<String> headers;
+    final String headersResponse;
+
+    public EchoBody() {
+        CRLF = "\r\n";
+        headers = List.of("POST");
+        headersResponse = String.format("Allow: %s", headers.get(0));
+    }
+
+
+    public String response(String method, String body) {
```

Comment:
Would like to see the interface implemented before merging.

Reply:
Added a `Route` interface which all routes now implement.